### ISSUE TITLE
Qdrant throws 'NoneType' object has no attribute 'CLOSED' when program exits

### ISSUE THREAD

**Issue by imneonizer** [qdrant/qdrant-client#58]
## Current Behavior

When the program exits, I see this error:
````
Exception ignored in: <function QdrantClient.__del__ at 0x7f6d7848e280>
Traceback (most recent call last):
  File "/anaconda3/lib/python3.8/site-packages/qdrant_client/qdrant_client.py", line 74, in __del__
  File "/anaconda3/lib/python3.8/site-packages/httpx/_client.py", line 1256, in close
  File "/anaconda3/lib/python3.8/site-packages/httpx/_transports/default.py", line 230, in close
  File "/anaconda3/lib/python3.8/site-packages/httpcore/_sync/connection_pool.py", line 307, in close
  File "/anaconda3/lib/python3.8/site-packages/httpcore/_sync/connection.py", line 159, in close
  File "/anaconda3/lib/python3.8/site-packages/httpcore/_sync/http11.py", line 216, in close
AttributeError: 'NoneType' object has no attribute 'CLOSED'
````

## Steps to Reproduce

````python
from qdrant_client import QdrantClient

client = QdrantClient(host="localhost", port=6333)
print(client.get_collections())
````

## Expected Behavior

Either the program should exit without any exceptions or I should be able to catch it.

## Context (Environment)

- I am using `qdrant-client==0.9.0` on `python 3.8`
- The quadrant server is run via docker compose
````
version: "3.8"

services:
  qdrant:
    image: qdrant/qdrant:v0.9.1
    container_name: qdrant_embedding_search
    ports:
      - 6333:6333
      - 6334:6334
    env_file:
      - ./.env
    volumes:
      - ./data:/qdrant/storage
      - ./config/qdrant.yaml:/qdrant/config/production.yaml
   ````

## Detailed Description

It seems the problem is with the `__del__` [implementation](https://github.com/qdrant/qdrant_client/blob/master/qdrant_client/qdrant_client.py#L71-L74]) when the program exists it tries to close the grpc and openapi connections but ends up throwing exception.

## Possible Implementation

A workaround I found is to close the connections manually before exiting the program:
````python
import time
import numpy as np
from qdrant_client import QdrantClient
from qdrant_client.http.models import Distance

client = QdrantClient(host="localhost", port=6333)
print(client.get_collections())

# Close connection manually
if client._grpc_channel is not None:
    client._grpc_channel.close()
client.openapi_client.client._client.close()
````

Please wrap the closing method inside try blocks and handle the exception or provide a `.close()` method for `QdrantClient` so the user can close it manually.

**Comment by generall** [comment]
Hi @imneonizer, thanks for reporting this!
I am currently trying to reproduce it using snippet you provided, but unfortunately I do not get the same result you get, so I the problem might be a bit deeper than that.

![image](https://user-images.githubusercontent.com/1935623/185786653-87fa6f24-863e-4e6b-b1e6-3e18bd1e4e4e.png)

**Comment by imneonizer** [comment]
Hey @generall, It seems like the problem is with my (base) Anaconda environment since I was building indexes outside the container.

**Comment by joein** [comment]
Any grpc connection closing issues should be closed with #177 

Should be available with the next `qdrant-client` release

Feel free to re-open the issue if it won't fix your problem
